### DISCUSSION TITLE
ActivityWatch available in `scoop` for Windows

### DISCUSSION THREAD

**Discussion by Pkcarreno** [ActivityWatch/activitywatch#894] [Show and tell]
Now you can install it through the command 

```
scoop install activitywatch
```

If you are a scoop user, you already know how it works.

**Comment by Pkcarreno** [comment]
Sorry if this is not the right place to say it, I don't know where else I could mention this.

**Comment by ErikBjare** [comment]
Very nice! I noticed this a few days ago on the [repology badge on the CI page](https://activitywatch.net/ci/).

Did you help out? If so, thanks! ❤️

**Reply by Pkcarreno** [reply]
I was surprised not to see ActivityWatch there, so I did my bit 👍
